azure-pipelines-1.yml

```diff
@@ -0,0 +1,45 @@
+# Python package
+# Create and test a Python package on multiple Python versions.
+# Add steps that analyze code, save the dist with the build record, publish to a PyPI-compatible index, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/python
+
+trigger:
+- test-package
```

Comment:
Does this need to be updated to dev or master for the trigger? Also, should we name this pipeline something descriptive, maybe `build-python-package-pipeline.yml` or similar?